Userland/Applications/FontEditor/GlyphEditorWidget.cpp

```diff
@@ -49,7 +49,7 @@ void GlyphEditorWidget::paint_event(GUI::PaintEvent& event)
     for (int x = 1; x < font().max_glyph_width(); ++x)
         painter.draw_line({ x * m_scale, 0 }, { x * m_scale, font().glyph_height() * m_scale }, palette().threed_shadow2());
 
-    auto bitmap = font().raw_glyph(m_glyph).glyph_bitmap();
+    auto bitmap = font().raw_glyph(m_glyph).value().glyph_bitmap();
```

Comment:
This is something I wanted to avoid because whether doing this is correct or not relies on state in the BitmapFont (whether it's unmasked or not) and since the API of GlyphEditorWidget isn't ensuring that the font you give it is unmasked, I can't assume that it is. Originally I wanted to add a separate type for unmasked fonts so the font editor can use those and assert it gets one because of the type but decided that would probably be too much.
I'm open to changing it though especially since the rest of the code still assumes it has an unmasked BitmapFont anyway so it doesn't really matter too much.